app-cdr/uif2iso/uif2iso-0.1.7c-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Convert CD images from uif (MagicISO) to iso"
+HOMEPAGE="http://aluigi.altervista.org/mytoolz.htm#${PN}"
```

Comment:
> And it also makes issues like https://github.com/gentoo/gentoo/pull/4718#issuecomment-303134168 a big waste of everyones time to solve what problem?

Not really a big waste of time, if I were to do that, I'd write a script to handle that for me.

> Maybe there's sense in arguing "for this package, the benefit is too small and only creates problems later", but the extrapolation of using that advice generically for the whole tree without thinking about it? I must disagree.

Most HOMEPAGE variable uses are really similar to this case. And there are very few ebuilds that do this: 245 total, ~20 of them are from dev-perl and prepend another homepage, i.e. `HOMEPAGE="http://example.com ${HOMEPAGE}"`.

I think that adding a variable in HOMEPAGE might discourage future maintainers to check it to see if it's dead or not. This is less of a problem with `SRC_URI` as it gets mirrored. It probably makes sense to add this check to `repoman` (as `HOMEPAGE.notfound` or something).